documentation/docs/30-advanced/30-link-options.md

```diff
@@ -66,6 +66,26 @@ Occasionally, we need to tell SvelteKit not to handle a link, but allow the brow
 
 Links with a `rel="external"` attribute will receive the same treatment. In addition, they will be ignored during [prerendering](page-options#prerender).
 
+## data-sveltekit-replacestate
+
+Sometimes you don't want navigation to create a new entry in the browser's session history. Adding a `data-sveltekit-replacestate` attribute to a link...
+
+```html
+<a data-sveltekit-replacestate href="/path">Path</a>
+```
+
+...will replace the current `history` entry rather than creating a new one with `pushState` when the link is clicked.
+
+## data-sveltekit-keepfocus
+
+Sometimes you don't want an element to lose focus after navigation. Adding a `data-sveltekit-keepfocus` attribute to a link...
+
+```html
+<a data-sveltekit-keepfocus href="/path">Path</a>
+```
+
+...will cause the currently focused element to retain focus after navigation. Otherwise, focus will be reset to the body.
```

Comment:
Note to self to try out: I'm wondering if this really keeps focus on the element prior to clicking the a tag, or if the a tag itself is then the focus. Probably the latter, which is why this mainly makes sense for `<form>` elements, which we probably should note here

Reply:
I'll give this a closer look later, but I think we should have some sort of note saying this is primarily intended for forms. I think it's usually an antipattern for links, unless you know what you're doing.

Reply:
Confirmed the currently-focused element is the link itself. I think the language is fine as-is, not sure there's a better way to phrase it that doesn't exclude the form case. I added a suggested paragraph below discussing the a11y considerations.